weblate/templates/snippets/list-objects.html

```diff
@@ -46,6 +46,16 @@
             </a>
           {% endif %}
         </th>
+        <th title="{% trans "Sort this column" %}" class="number zero-width-640 sort-cell">
+          {% if objects.paginator.num_pages > 1 %}
+            <a href="?page={{ objects.number }}&amp;sort_by={% if objects.paginator.sort_by == "approved" %}-{% endif %}approved">
+          {% endif %}
+          {% trans "Approved" %}
+          <span class="sort-icon {% if objects.paginator.sort_by == "approved" %}sort-down{% elif objects.paginator.sort_by == "-approved" %}sort-up{% endif %}" />
+          {% if objects.paginator.num_pages > 1 %}
+            </a>
+          {% endif %}
+        </th>
```

Comment:
This column should be shown only if reviews are turned on, it will show just useless zeroes when turned off.